### DISCUSSION TITLE
Icon-offset v.s. icon-translate

### DISCUSSION THREAD

**Discussion by xinyuluo** [maplibre/maplibre-gl-js#1114] [Q&A]
Hi! I wonder what's the difference between icon-offset and icon-translate in symbol layer? 
https://maplibre.org/maplibre-gl-js-docs/style-spec/layers/#layout-symbol-icon-offset
To me, it feels like the two achieve the same thing.

Thank you!

**Comment by wipfli** [comment]
Maybe they are the same with `symbol-placement: point` but different for `symbol-placement: line`?

**Comment by drwestco** [comment]
Pretty sure `icon-translate` is a paint-only effect, and isn't taken into account for position + collision calculations.
